carbon/assetlist.json

```diff
@@ -219,6 +327,18 @@
         "name": "USD Coin (BEP-20)",
         "display": "usdc",
         "symbol": "USDC",
+        "traces": [{
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "binancesmartchain",
+            "base_denom": "USDC",
```

Comment:
this base_denom should be a contract address

Reply:
maybe this?  (really not sure exactly which one--there are so many different options)
0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d